infra/k8s/testground-infra/values.yaml

```diff
@@ -17,6 +17,9 @@
 #   See https://github.com/helm/charts/blob/master/stable/prometheus-operator/README.md#L193
 #   for an explanation of this option.
 prometheus-operator:
+  podAnnotations: {
+    cni: "flannel"
+  }
   alertmanager:
     enabled: false
   grafana:
```

Comment:
I don't see how to move Grafana to the control network, but at least test plans don't need access to it.